src/plugins/discover/public/application/angular/doc_table/create_doc_table_react.tsx

```diff
@@ -130,5 +135,31 @@ export function DocTableLegacy(renderProps: DocTableLegacyProps) {
       return renderFn(ref.current, renderProps);
     }
   }, [renderFn, renderProps]);
-  return <div ref={ref} />;
+  return (
+    <div>
+      <div ref={ref} />
+      {renderProps.rows.length === renderProps.sampleSize ? (
+        <div
+          className="dscTable__footer"
+          data-test-subj="discoverDocTableFooter"
+          tabIndex={-1}
+          id="discoverBottomMarker"
+        >
+          <FormattedMessage
+            id="discover.howToSeeOtherMatchingDocumentsDescription"
+            defaultMessage="These are the first {sampleSize} documents matching
+                  your search, refine your search to see others."
+            values={{ sampleSize: renderProps.sampleSize }}
+          />
+          <EuiButtonEmpty onClick={renderProps.onBackToTop}>
+            <FormattedMessage id="discover.backToTopLinkText" defaultMessage="Back to top." />
+          </EuiButtonEmpty>
+        </div>
+      ) : (
+        <span tabIndex={-1} id="discoverBottomMarker">
+          &#8203;
+        </span>
```

Comment:
Could you explain me shortly, why we need an empty element here?

Reply:
I was a hack when the scroll to bottom function was deangularized, to make it work in Safari, right dear @dej611?

Reply:
@timroes from my tests Safari required an element with some content in it to trigger a scroll. That element is the minimal set to make Safari scroll at the end of the table.